spyder/plugins/ipythonconsole.py

```diff
@@ -1530,6 +1545,18 @@ def create_kernel_spec(self, is_cython=False):
                      self.main.get_spyder_pythonpath())
         return SpyderKernelSpec(is_cython=is_cython)
 
+    def get_stderr_file_handle(self, stderr_file):

```

Comment:
To avoid repeated code between this and the method you added in `widgets/ipythonconsole/client.py`, I think we should move this to `spyder/utils/misc.py`. Fortunately, it doesn't depend on any other method of `ipythonconsole.py` or `client.py`, so that should work without problems.

Reply:
My fault. I have thought about moving that functionality to a common place but later a I forget about it. Done!